git.md

```diff
@@ -198,9 +198,41 @@ Where relevant, go through the changes in deploy preview or in your local enviro
 
 Prioritize on reviewing those that marked as bug fixes, then the new features. Use `G2M` as a search filter to minimize noise -- provided that it is a well-followed convention in the given project.
 
+## On Github Projects
```

Comment:
this is not core `git` related (pull/merge request section is an exception). we should move this to either:

- under `communications/`
- or, a section of its own, under perhaps a larger group (such as "Project Planning/Management".

Reply:
I thought about having its own file too. About screenshots can be done!